templates/common/powervs/units/powervs-hostname.service.yaml

```diff
@@ -9,10 +9,13 @@ contents: |
   After=NetworkManager-wait-online.service
   # Run before hostname checks
   Before=node-valid-hostname.service
+  # We require afterburn to have run first
+  Wants=afterburn.service
```

Comment:
We want a Requires here, that way the unit would be explicitly marked failed because its dependency failed.